### DISCUSSION TITLE
How do I use Dark Reader's Api on my website?

### DISCUSSION THREAD

**Discussion by Elrigor** [darkreader/darkreader#9397] [Q&A]
Hi!

I'd like to use DarkReader's API to make a Dark/Light switch for my website, however, I'm new with NPM and I don't really know what to do 😅...

I installed NPM via apt on Ubuntu 20.04 and then ran this two commands:

```
npm install darkreader
npm run api
```

But it breaks in the first one because a **not found package.json**
Can someone please help me to set Dark Reader's API up. please?
Thank you so much!

**Comment by alexanderby** [comment]
If you never used NPM before, then you may try downloading or adding a link to something like UNPKG:
```html
<head>
    <!-- Link to Dark Reader -->
    <script src="https://unpkg.com/darkreader@4.9.52/darkreader.js"></script>
    <!-- Run Dark Reader script with default settings -->
    <script>
        DarkReader.enable();
    </script>
</head>
```

**Reply by Elrigor** [reply]
Thank you so much, that helped me a lot!
